.snippets/code/builders/interoperability/xcm/send-execute-xcm/dry-run/dry-run-xcm.js

```diff
@@ -0,0 +1,62 @@
+import { ApiPromise, WsProvider } from '@polkadot/api';
+
+const main = async () => {
+  try {
+    // Construct API provider
+    const wsProvider = new WsProvider('INSERT_WSS_ENDPOINT');
+    const api = await ApiPromise.create({ provider: wsProvider });
+    console.log('Connected to the API. Preparing dry run XCM call...');
+
+    // Define the origin
+    const origin = { V4: { parents: 1, interior: 'Here' } };
+
+    const assetMultiLocation = {
+      parents: 0,
+      interior: { X1: { PalletInstance: 3 } },
+    }; // The asset's location (adjust PalletInstance as needed)
+
+    const amountToSend = 1000000000000; // Adjust this value as needed
+
+    const message = {
+      V4: [
+        {
+          WithdrawAsset: [
+            {
+              id: { parents: 1, interior: 'Here' },
+              fun: { Fungible: amountToSend },
+            },
+          ],
+        },
+        {
+          BuyExecution: {
+            fees: {
+              id: { parents: 1, interior: 'Here' },
+              fun: { Fungible: amountToSend },
+            },
+            weightLimit: { Unlimited: null },
+          },
+        },
+        {
+          ClearOrigin: null,
```

Comment:
Im curious as to this XCM message that you put here. There is no Deposit Asset so this would actually result in AssetsTrapped right?